xmtp/src/client.rs

```diff
@@ -122,17 +127,19 @@ mod tests {
             .unwrap();
 
         assert_eq!(contacts.len(), 1);
-        assert!(contacts[0].prekey.is_some());
-        assert!(contacts[0].identity_key.is_some());
+        contacts[0].vmac_identity_key();
+        contacts[0].vmac_fallback_key();
```

Comment:
what are we hoping to check here?

Reply:
That it doesn't panic. There are a bunch of layers of unwraps happening under the hood